docs/getting-started.rst

```diff
@@ -32,9 +32,9 @@ To generate a new role with Molecule, simply run:
 
 .. code-block:: bash
 
-    $ molecule init role my-new-role --driver-name docker
+    $ molecule init role my_new_role --driver-name docker
 
-You should then see a ``my-new-role`` folder in your current directory.
+You should then see a ``my_new_role`` folder in your current directory.
 
 .. note::
 
```

Comment:
I suppose it could be fixed in the note below as well.

Reply:
Thank you for quick review. I fixed it.